### PR TITLE
[SRVCOM-1284] Update Serving and Eventing CRDs

### PR DESCRIPTION
**Pull request by nak3**
This patch updates `operator_v1alpha1_knativeeventing_crd.yaml` and `operator_v1alpha1_knativeserving_crd.yaml` that are very old.

### CODE REVIEW

**Comment by nak3** [comment]
/test 4.7-operator-e2e-aws-ocp-47

I think infra issue.

File: olm-catalog/serverless-operator/hack/update-manifests.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```
Comment (markusthoemmes) [inline olm-catalog/serverless-operator/hack/update-manifests.sh:1]: Are we calling this from anywhere? Should we add it to the `Makefile` so it's automatically taken into account?

Comment (matzew) [reply olm-catalog/serverless-operator/hack/update-manifests.sh:1]: Upstream they were already updated, so I am suprised we have old files here.

Perhaps update the script to pull em from upstream?

Comment (nak3) [reply olm-catalog/serverless-operator/hack/update-manifests.sh:1]: Sure, added it to the Makefile.

And yes this script will pull them from upstream :+1:

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**